src/api/axios.js

```diff
@@ -0,0 +1,162 @@
+
+  return res;
+}
+
+async function updateUser(userId) {
+  const res = await axios.patch(
+    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/users/${userId}`
+  );
+
+  return res;
+}
+
+async function updateCounselor(payload) {
+  const res = await axios.patch(
+    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/users/${payload.userId}`,
+    payload.newCounselorInfo,
+    {
+      withCredentials: true,
+    }
+  );
+
+  return res;
+}
+
+async function updateCounselorSchedule(payload) {
+  const res = await axios.post(
+    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/users/${payload.userId}/counselor/availableDates`,
+    payload.availableDates,
+    {
+      withCredentials: true,
+    }
+  );
+
+  return res;
+}
+
+async function deleteCounselorSchedule(userId, id) {
+  const res = await axios.delete(
+    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/users/${userId}/counselor/availableDates/${id}`,
+    {
+      withCredentials: true,
+    }
+  );
+
+  return res;
+}
+
+async function getCounselList(options) {
+  const res = await axios.get(
+    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/counsels`,
+    options
+  );
+
+  return res;
+}
+
+async function getCounsel(counsel_id) {
+  const res = await axios.get(
+    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/counsels/${counsel_id}`,
+    {
+      withCredentials: true,
+    }
+  );
+
+  return res;
+}
+
+async function getStory(userId) {
+  const res = await axios.get(
+    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/counsels/${userId}`
+  );
+
+  return res;
+}
+
+async function getReservedCounselList(type, userId, page) {
+  const res = await axios.get(
+    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/counsels/reserved`,
+    {
+      params: {
+        [type === "counselee" ? "counselee" : "counselor"]: userId,
+        page,
+      },
+      withCredentials: true,
+    }
+  );
+
+  return res;
+}
+
+async function getCounselorSchedule(counselor, user_id) {
+  const res = await axios.get(
+    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/counsels/schedules?counselor=${counselor._id}&counselee=${user_id}`,
+    {
+      withCredentials: true,
+    }
+  );
+
+  return res;
+}
+
+async function createCounsel(newStory) {
+  const res = await axios.post(
+    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/counsels`,
+    newStory,
+    {
+      withCredentials: true,
+    }
+  );
+
+  return res;
+}
+
+async function updateCounselors(counsel_id, userId) {
+  const res = await axios.post(
+    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/counsels/${counsel_id}/counselors`,
+    {
+      userId,
+    },
+    {
+      withCredentials: true,
+    }
+  );
+
+  return res;
+}
+
+async function updateCounsel(counsel_id, user_id) {
+  const res = await axios.post(
+    `${process.env.REACT_APP_LOCAL_SERVER_URL}/api/counsels/${counsel_id}/counselors/${user_id}`,
+    {
+      withCredentials: true,
+    }
+  );
+
+  return res;
+}
+
+export {
+  getCounselor,
+  updateUser,
```

Comment:
```suggestion
export default {
  getCounselor,
  updateUser,
```
하나의 객체에 담아서 보내므로 `export default` 로 사용하면 좋을 것 같아요~!

Reply:
export default가 안되서 빼줬는데 함수를 여러개 보내기 때문에 안 되는 것 같아요... [모듈 내보내고 가져오기](https://ko.javascript.info/import-export)

Reply:
그렇군요! 저도 다시 확인해보니 리스트를 보낼때는 export로 사용을 하도록 나와있네요~ 감사합니다